atd-vzv/src/views/map/Map.js

```diff
@@ -27,20 +27,9 @@ import "mapbox-gl/dist/mapbox-gl.css";
 import "@mapbox/mapbox-gl-draw/dist/mapbox-gl-draw.css"; // Get out-of-the-box icons
 import MapInfoBox from "./InfoBox/MapInfoBox";
 import MapPolygonInfoBox from "./InfoBox/MapPolygonInfoBox";
+import MapGeocoder from "./Geocoder/Geocoder";
 
-const MAPBOX_TOKEN = `pk.eyJ1Ijoiam9obmNsYXJ5IiwiYSI6ImNrM29wNnB3dDAwcXEzY29zMTU5bWkzOWgifQ.KKvoz6s4NKNHkFVSnGZonw`;
-
-function useMapEventHandler(eventName, callback, mapRef) {
-  useEffect(() => {
-    const currentMapRef = mapRef.current;
-    const mapDataListener = currentMapRef.on(eventName, function () {
-      callback();
-    });
-    return () => {
-      currentMapRef.off(eventName, mapDataListener);
-    };
-  }, [eventName, callback, mapRef]);
-}
```

Comment:
Moved this to the helpers file and added a docstring because it was a quick change